Controller/Adminhtml/Order/CreateAndPrintMyParcelTrack.php

```diff
@@ -115,25 +116,29 @@ private function massAction()
             return $this;
         }
 
-        $this->orderCollection
-            ->setMagentoTrack()
-            ->setMyParcelTrack()
-            ->createMyParcelConcepts()
-            ->updateGridByOrder();
-
-        if (
-            $this->orderCollection->getOption('request_type') == 'concept' ||
-            $this->orderCollection->myParcelCollection->isEmpty()
-        ) {
-            return $this;
-        }
+        if (TrackTraceHolder::EXPORT_MODE_PPS === $this->orderCollection->getExportMode()) {
```

Comment:
deze constante lijkt me niet logisch in de TrackTraceHolder, is er geen settingsdata class? Ik zou hem hier niet verwachten... maar dat kan ook aan mij liggen in dit geval